Review slh updates (#196)
w3c/wai-website-theme: _style-guide/grammar.md

## Punctuation

> #### Semicolon

Reviewer: ```suggestion
### Semicolon
```

> See [Semicolon (optional) in Syntax for related information in sentences](/style-guide/grammar/#semicolon-optional).

Reviewer: ```suggestion
See [Semicolon (optional) in Related information in sentences](/style-guide/structure-presentation/#semicolon-optional).
```

## Punctuation

> <s> <!-- I think delete the colon section. For simplicity and consistency, I want us to use em dashes for list items. Then this does not provide information beyond standard grammar. And I'm guessing the mis-use of colon is not a common error in WCAG 3 writing? -->

Reviewer 2: Can this be actioned?

Author: @iadawn and @shawna-slh - 

>
>I think delete the colon section. For simplicity and consistency, I want us to use em dashes for list items. Then this does not provide information beyond standard grammar. And I'm guessing the mis-use of colon is not a common error in WCAG 3 writing? 
>

I have removed the guidance for colons, as requested.

I do want to note that previous discussions have identified different use cases for colons and em dashes in lists, so they are not always interchangeable. For reference:

- [#30 Colons versus em dashes in list items](https://github.com/w3c/style-guide-v2025/issues/30#issuecomment-3988654859)
- [#31 When to use an em dash and when to use a colon](https://github.com/w3c/style-guide-v2025/issues/31#issuecomment-3988562137)

Given that, my understanding is that using em dashes as the default approach would work in many cases, but there may be some scenarios where a colon is more appropriate.

There are also some inconsistencies in how both are currently used across WAI materials, which we may want to keep in mind as we aim for clarity and consistency.

#### Em dash (—) {#em-dash}

Put a space before and after an em dash.

> For em dashes in sentences, see [Em dash (optional) in Syntax for related information in sentences](/style-guide/grammar/#em-dash-optional).

Reviewer: ```suggestion
For em dashes in sentences, see [Em dash (optional) in Related information in sentences](/style-guide/structure-presentation/#semicolon-optional).
```

> For em dashes in sentences, see [Em dash (optional) in Related information in sentences](/style-guide/structure-presentation/#semicolon-optional).

Reviewer: ```suggestion
For em dashes in sentences, see [Em dash (optional) in Related information in sentences](/style-guide/structure-presentation/#em-dash-optional).
```

### Parentheses

> See [Parentheses (optional) in Syntax for related information in sentences](/style-guide/grammar/#parentheses-optional).

Reviewer: ```suggestion
See [Parentheses (optional) in Related information in sentences](/style-guide/structure-presentation/#semicolon-optional).
```

> See [Parentheses (optional) in Related information in sentences](/style-guide/structure-presentation/#semicolon-optional).

Reviewer: ```suggestion
See [Parentheses (optional) in Related information in sentences](/style-guide/structure-presentation/#parentheses-optional).
```
